Automattic/wp-calypso
# ESLint: Deprecate Lodash methods we currently do not use

#### Changes proposed in this Pull Request
 
This PR deprecates some Lodash methods we currently do not use and have native JS replacements:

* `all`
* `any`
* `collect`
* `contains`
* `detect`
* `extendOwn`
* `foldl`
* `foldr`
* `inject`
* `lastIndexOf`
* `padEnd`
* `padStart`
* `select`

There are more we could add, but the plan is to do this incrementally and in small steps. 

#### Testing instructions

* Try out importing each of the lodash methods.
* Verify you're getting an ESLint error.
* Verify these aren't used anywhere at this time (there should be no new ESLint errors introduced).
